PROPOSALS.md

```diff
@@ -88,6 +88,7 @@ request feedback on feasibility before investing time and effort in the full pro
     * Make sure to include a link to the CLC discussion into the commit message.
     * Make sure to update `libraries/base/changelog.md`. (You might like to insert your new entry
       at an arbitrary position instead of the last one to minimise merge conflicts.)
+    * Raise an issue, linking to the GitHub proposal (this is needed for assigning milestones)
```

Comment:
I think this addition is redundant now once you added it to the post-approval list.